src/main/java/com/cinema/model/ShoppingCart.java

```diff
@@ -0,0 +1,76 @@
+package com.cinema.model;
+
+import java.util.List;
+import java.util.Objects;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.MapsId;
+import javax.persistence.OneToMany;
+import javax.persistence.OneToOne;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "shopping_cart")
+public class ShoppingCart {
+    @Id
+    @Column(name = "shopping_Cart_id")
```

Comment:
```suggestion
    @Column(name = "shopping_cart_id")
```

also, do you use this annotation at all? you seem to have the only `id` column in your table (created by `@MapsId`